README.md

```diff
@@ -1183,6 +1183,33 @@ Prints and Writes now time + # minutes.
       Var: Result Var with now time + # minutes
       File: File path (Optional)
 
+### <a id="generate_unique_number"></a>generate_unique_number
+
+Returns a variable with a unique number generated at the end of it.
+The method receives "Carlos" and then returns "Carlos~452~"
+
+    - Type: generate_unique_number
+      Name: number that user needs to be unique
+      ResultVar: Var that will have the unique number value
+
+### <a id="generate_unique_email"></a>generate_unique_email
+
+Returns a variable with a unique email using ramdon string at the begining 
+method receives "@gmail.com" and then returns "ujrb@gmail.com"
+
+    - Type: generate_unique_email
```

Comment:
This description is not needed anymore as the method was removed

Reply:
Done